src/components/com_tjcertificate/administrator/models/certificates.php

```diff
@@ -81,18 +88,63 @@ protected function getListQuery()
 		// Initialize variables.
 		$db    = $this->getDbo();
 		$query = $db->getQuery(true);
-		$app = Factory::getApplication();
+		$app   = Factory::getApplication();
+		$user  = Factory::getUser();
 
 		$extension = Factory::getApplication()->input->get('extension', '', 'CMD');
 
 		$this->setState('filter.component', $extension);
 
+		// Filter by client
+		$client = $this->getState('filter.client');
+
 		// Create the base select statement.
 		$query->select(array('ci.*', 'ct.title', 'users.name as uname'));
 		$query->from($db->quoteName('#__tj_certificate_issue', 'ci'));
+
 		$query->join('LEFT', $db->quoteName('#__tj_certificate_templates', 'ct') .
 			' ON (' . $db->quoteName('ci.certificate_template_id') . ' = ' . $db->quoteName('ct.id') . ')');
-		$query->join('LEFT', $db->quoteName('#__users', 'users') . ' ON (' . $db->quoteName('ci.user_id') . ' = ' . $db->quoteName('users.id') . ')');
+
+		$query->join('LEFT', $db->quoteName('#__users', 'users') .
+			' ON (' . $db->quoteName('ci.user_id') . ' = ' . $db->quoteName('users.id') . ')');
+
+		if (ComponentHelper::isEnabled($this->multiagency) && $this->params->get('enable_multiagency'))
+		{
+			$canManageAllAgencyUser = $user->authorise('core.manage.all.agency.user', $this->multiagency);
```

Comment:
write step-by-step code...use this statement just before it is being used...so that it is logically at the right place while debugging...put this before the line 129

Reply:
done